lib/client.js

```diff
@@ -357,22 +391,42 @@ extend(Raven.prototype, {
       res.sentry = eventId;
       return next(err);
     };
+  },
+
+  captureBreadcrumb: function (breadcrumb) {
+    if (!this.installed) return;
+
+    breadcrumb = extend({
+      timestamp: +new Date / 1000
+    }, breadcrumb);
+
+    var currCtx = this.getContext();
+    if (!currCtx.breadcrumbs) currCtx.breadcrumbs = [];
+    currCtx.breadcrumbs.push(breadcrumb);
+    if (currCtx.breadcrumbs.length > this.maxBreadcrumbs) {
```

Comment:
Per our conversation, I'd probably err on recording, say, 30 breadcrumbs per context and allow users to increase it to the max if they really want to. Better safe than sorry.